src/workerd/io/worker.h

```diff
@@ -638,10 +638,22 @@ class Worker::Actor final: public kj::Refcounted {
 
   using Id = kj::OneOf<kj::Own<ActorIdFactory::ActorId>, kj::String>;
 
+  class Loopback {
```

Comment:
bikeshedding: Is this really a loopback? It kinda feels like a factory type for a specific actor.

(Copied from last PR)

Reply:
That was my initial reaction also. The name Loopback just doesn't feel right here. But consider that non-blocking